BUILDING_ON_WINDOWS.md

```diff
@@ -95,6 +121,7 @@ Open up your terminal with the Visual Studio environment variables (e.g. `x64 Na
 1. `cd build`
 1. `conan install .. -s build_type=Release -c tools.cmake.cmaketoolchain:generator="NMake Makefiles" --build=missing --output-folder=.`
 1. `cmake -G"NMake Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE="conan_toolchain.cmake" -DCMAKE_PREFIX_PATH="C:\Qt\5.15.2\msvc2019_64" ..`
+1. `cd ..`
```

Comment:
If I recall correctly, it wouldn't let me use 'nmake' in the build folder. I had to be in the chatterino2 folder for it to work. 

Right now, when I try to do 'nmake' in the build folder, I get 
'NMAKE : fatal error U1064: MAKEFILE not found and no target specified
Stop.'

However, I'm not managing to build successfully at the moment (I keep getting LINK and NMAKE fatal errors that I have not currently figured out) even with 'nmake' in the chatterino2 folder because I get 'fatal error' for LINK and NMAKE. 

My best guess is because I messed up something with either the GitHub branches, git commands, and/or something in VS Code (which I use over Visual Studio Community right now) when I tried to get the copy of the original repository in my fork without being 'commits ahead'.

Reply:
> Right now, when I try to do 'nmake' in the build folder, I get
'NMAKE : fatal error U1064: MAKEFILE not found and no target specified
Stop.'

I usually use ninja as the generator, but this shouldn't be different to nmake. From my experience, this happens when `CMake` fails to configure the project correctly.

For a debug build, the following should be enough. Make sure you remove the `build/CMakeCache.txt` if you want a clean reconfiguration.
```
chatterino2/build> conan install .. --output-folder=. -b missing -c tools.cmake.cmaketoolchain:generator=Ninja -s build_type=Debug
chatterino2/build> cmake -G"NMake Makefiles" -DCMAKE_TOOLCHAIN_FILE="conan_toolchain.cmake" -DCMAKE_PREFIX_PATH="C:\Qt\5.15.2\msvc2019_64" ..
```

> I keep getting LINK and NMAKE fatal errors that I have not currently figured out

Linker errors shouldn't be related to the build folder you're in. If you can paste the error here, we might be able to figure this out.

Reply:
Ok I used the two commands you listed
`chatterino2/build> conan install .. --output-folder=. -b missing -c tools.cmake.cmaketoolchain:generator=Ninja -s build_type=Debug`
and
`chatterino2/build> cmake -G"NMake Makefiles" -DCMAKE_TOOLCHAIN_FILE="conan_toolchain.cmake" -DCMAKE_PREFIX_PATH="C:\Qt\5.15.2\msvc2019_64" ..`
and I deleted the CMakeCache.txt file and it 'Built target chatterino'.

I also tried the regular commands I used before (in the 'BUILDING_ON_WINDOWS.md' file) and it built successfully so I don't have linker errors anymore. Maybe because of the deleted CMakeCache.txt file?

Thanks for the help in fixing my error.

(Also the 'nmake' works in the build directory now so there's no need for 'cd ..').

Reply:
I removed the 'cd ..'

Reply:
> Maybe because of the deleted CMakeCache.txt file?

That might have been the case. When you switch branches (or similar) that had more complex changes to the `CMakeLists.txt` (i.e. not just adding files), then CMake can sometimes get confused as old values from `CMakeCache.txt` aren't up-to-date anymore.

Reply:
Ah okay, thanks for the information.